### ISSUE TITLE
Fork for eWaterCycle forcing compliance

### ISSUE THREAD

**Issue by BSchilperoort** [Daafip/HBV-bmi#18]
@RolfHut and I were working with the HBV model and noticed some points where `ewatercycle-hbv` and `HBV-bmi` did not fully fit in the eWaterCycle philosophy.

MakkinkForcing was a valid input, but `ewatercycle-hbv` would write new files during `model.setup()`. This is not good for reproducibility, and doesn't allow users to use forcing on a read-only disk.

As the workshop starts Monday, I made changes in a fork of this repo + ewatercycle-hbv; 
- https://github.com/eWaterCycle/HBV-bmi
- https://github.com/eWaterCycle/ewatercycle-hbv

The HBV model now expects input in SI (`K` and `kg m-2 s-1`), and converts these to degrees Celcius and mm/d. This means that CaravanForcing and LumpedMakkinkForcing are directly compatible with HBV.

Note that I disabled the "HBVForcing" entrypoint: the way HBVForcing is structured needs to be reworked slightly, but I did not have any time for this now.

It would be nice to merge these changes back into this repo (and ewatercycle-hbv) at a later point in time.

**Comment by Daafip** [comment]
I saw your initial changes but wanted to take the time to review them properly. Id be happy for ewatercycle-HBV to continue to develop so do look into merging changes back into the main repo.

**Comment by BSchilperoort** [comment]
> I saw your initial changes but wanted to take the time to review them properly

Yeah I assumed you would not have time and we need things to work this week. I'll clean things up and make a PR at some point.
